sdk/keyvault/tests.yml

```diff
@@ -10,13 +10,12 @@ resources:
 jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
-      MaxParallel: 1
+      MaxParallel: 5
```

Comment:
Should've thought of this earlier. We can remove `MaxParallel` here because the [default is 0](https://github.com/Azure/azure-sdk-for-python/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L4) (which means fully parallel). 

One would only want to set `MaxParallel` when wanting to constrain the parallelism below the number of entries in the matrix (like to 1 to solve race condition problems).